src/components/reporting/reporting.resource.tsx

```diff
@@ -14,6 +14,15 @@ type dynamicReportRequest = {
   endDate: string;
 };
 
+type saveReportRequest = {
+  reportName: string;
+  reportDescription?: string;
+  reportType?: string;
+  columns?: string;
+  rows: string;
+  report_request_object: string;
+};
+
 export function useReports(params: fixedReportRequest) {
   const apiUrl = `/ws/rest/v1/ugandaemrreports/reportingDefinition?uuid=${params.reportUUID}&startDate=${params.startDate}&endDate=${params.endDate}`;
```

Comment:
use the openmrs -ws-rest-api-context to make api calls instead of defining your api-context

Reply:
@akileng56 add a ticket to your PR.

Reply:
Done